.circleci/config.yml

```diff
@@ -17,11 +17,11 @@ parameters:
     default: "solbuildpackpusher/solidity-buildpack-deps@sha256:7f0b62da2aa0920660e67cd17fb3a7f6c57d7da72454003056f3bd27bbda2075"
   ubuntu-2204-clang-docker-image:
     type: string
-    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2204.clang-2
-    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:0d4fa1da806f9b3daabd2d1dc0842899af794a838e475c14045c3fb289cc8103"
-  ubuntu-1604-clang-ossfuzz-docker-image:
+    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2204.clang-1
+    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:e515710752bfb38ee0e9b2f92ca71612c03ed85290c36c3b3c0a3d9f90187aef"
+  ubuntu-clang-ossfuzz-docker-image:
```

Comment:
This apparently goes back to a previous version of an image? What's happening here?

Reply:
I updated it to 2 because this PR https://github.com/ethereum/solidity/commit/3e1e43f5691f2bb05ef2c64c99792ff6b181f769 updated it while I was doing the refactor. So I had a merge conflict and kept the latest version but applied the renaming. I didn't bump the `ubuntu2204-clang` because there was not significant change in the image, only comments and the Dockerfile renaming.